bitcoin/src/blockdata/transaction.rs

```diff
@@ -122,6 +122,7 @@ impl OutPoint {
 
     /// Creates a new [`OutPoint`].
     #[inline]
+    #[deprecated(since = "TBD", note = "specify concrete values for each field instead")]
```

Comment:
Makes it sound like the values here are not concrete. I'd say something like "use the struct initialization syntax instead"

Reply:
I spent some time searching docs when I wrote this and I could not find a definitive place that defined "struct initialization syntax" - is that a standard term? Its mainly just out of interest. Since I wrote the same phrase originally, you wrote it now, and @apoelstra wrote something similar elsewhere I'll go ahead and use it.

Reply:
I don't know but that's what came to my mind when I was trying to find a better way to convey it. Searching did occur to me but I was too lazy. :)